docs/firebase.md

```diff
@@ -91,31 +91,6 @@ import { onCallGenkit } from 'firebase-functions/https';
 export generatePoem = onCallGenkit(generatePoemFlow);
 ```
 
-### Define an authorization policy
```

Comment:
What should we be telling developers to do instead?

Reply:
We are drafting a solution that aligns better with existing Genkit SDK (ContextProvider), and I suggest we omit this section until that solution is fully hashed and released.